components/dashboard/src/repositories/detail/workspaces/WorkpaceSizeOptions.tsx

```diff
@@ -56,31 +51,14 @@ export const ConfigurationWorkspaceSizeOptions = ({ configuration }: Props) => {
         [configuration.id, selectedValue, toast, updateConfiguration],
     );
 
-    if (isError || !classes) {
-        return <div>Something went wrong</div>;
-    }
-
-    if (isLoading) {
-        return <LoadingState />;
-    }
-
     return (
         <ConfigurationSettingsField>
             <form onSubmit={setWorkspaceClass}>
                 <Heading3>Workspace Size Options</Heading3>
                 <Subheading>Choose the size of your workspace based on the resources you need.</Subheading>
 
-                <RadioGroup value={selectedValue} onValueChange={setSelectedValue} className="mt-4">
-                    {classes.map((wsClass) => (
-                        <Label className="flex items-start space-x-2 my-2">
-                            <RadioGroupItem value={wsClass.id} />
-                            <div className="flex flex-col space-y-2">
-                                <span className="font-bold">{wsClass.displayName}</span>
-                                <span>{wsClass.description}</span>
-                            </div>
-                        </Label>
-                    ))}
-                </RadioGroup>
+                <WorkspaceClassOptions value={selectedValue} onChange={setSelectedValue} />
```

Comment:
Thanks for componentizing this! It makes total sense and it will help when we update the user settings to this new pattern.